_search-plugins/star-tree-index.md

```diff
@@ -39,26 +39,26 @@ The following image illustrates a standard star-tree index structure.
 
 <img src="{{site.url}}{{site.baseurl}}/images/star-tree-index.png" alt="A star-tree index containing two dimensions and two metrics" width="700">
 
-Sorted and aggregated star-tree documents are backed by `doc_values` in an index. `doc_values` use the following pattern:
+Sorted and aggregated STIX documents are backed by `doc_values` in an index. The columnar data found in `doc_values` are stored using the following properties:
 
-- The values are sorted based on the order of their `ordered_dimension`. In the preceding image, the dimensions are determined by the `status` setting and then by the `port` for each status.
+- The values are sorted based on the fields set in the `ordered_dimension` setting. The first field configured in `ordered_dimensions` are considered primary fields, and any corresponding fields of the primary are considered secondary.  In the preceding image, the dimensions are determined by the `status` setting and then by the `port` for each status.
```

Comment:
Maybe we can just remove the following line ? since the image explanation might give the clarity the user needs ?

"""
The first field configured in `ordered_dimensions` are considered primary fields, and any corresponding fields of the primary are considered secondary. 
"""